indexers/vector/SptagIndexer/Dockerfile

```diff
@@ -12,10 +12,8 @@ RUN wget https://cmake.org/files/v3.15/cmake-3.15.5.tar.gz && \
     make -j2 && \
     make install
 
-RUN git clone https://github.com/microsoft/SPTAG.git
+RUN git clone https://github.com/microsoft/SPTAG.git/tree/a91ed49a041a24bd145874d3b3c815bde0430e65
```

Comment:
can we merge these 2 RUN layers?

Reply:
I thought so but have to reverse the commit